Dockerfile.proto

```diff
@@ -72,6 +76,11 @@ RUN wget https://github.com/pseudomuto/protoc-gen-doc/releases/download/v${GFPRO
 	tar xzvf protoc-gen-doc_${GFPROTOCGENDOC}_linux_amd64.tar.gz && \
 	mv protoc-gen-doc /usr/local/bin
 
+##
+## Install Google AIP api-linter
```

Comment:
Nit: AIP -> API

Reply:
Yeah I thought that too when I was reading the documentation, but it is AIP. AIP stands for API Improvement Proposals. I really wish they had chosen a different name 😵‍💫

Reply:
Oh interesting :smile: I was reviewing on my phone and saw that first... will take a closer look today.